easybuild/easyconfigs/h/hisat/hisat-0.1.5-beta-goolf-1.7.20.eb

```diff
@@ -0,0 +1,31 @@
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
```

Comment:
toolchain should remain below `description`, above `sources` (with an empty line in between); yes, I know I'm being picky, but enforcing this style makes it easy to read other easyconfigs for everyone